### PR TITLE
misc ivy fixes #2

### PR DESCRIPTION
**Pull request by IgorMinar**

### CODE REVIEW

**Review by kara — APPROVED** [review]
LGTM

File: packages/compiler-cli/integrationtest/bazel/injectable_def/app/test/BUILD.bazel [hunk 1]
```diff
@@ -31,6 +31,7 @@ jasmine_node_test(
         ":test_lib",
```
Comment (kara) [inline packages/compiler-cli/integrationtest/bazel/injectable_def/app/test/BUILD.bazel]: Same here, is this ready to remove `fixme-ivy-aot`?

**Review by kara — DISMISSED** [review]

File: packages/compiler-cli/integrationtest/bazel/injector_def/ivy_build/app/test/BUILD.bazel [hunk 2]
```diff
@@ -21,6 +25,7 @@ jasmine_node_test(
     bootstrap = ["angular/tools/testing/init_node_spec.js"],
     tags = [
         "fixme-ivy-aot",
```
[the remark this answers is not in the document]
Comment (IgorMinar) [reply packages/compiler-cli/integrationtest/bazel/injector_def/ivy_build/app/test/BUILD.bazel]: these tests don't pass yet. I have another fix in works that will make these tests pass.

File: packages/compiler-cli/integrationtest/bazel/injectable_def/app/test/BUILD.bazel (hunk 1 above)
Comment (IgorMinar) [reply packages/compiler-cli/integrationtest/bazel/injectable_def/app/test/BUILD.bazel]: same as above

**Review by kara — APPROVED** [review]

**Pull request closed without merging**